uuid-registry.txt

```diff
@@ -0,0 +1,158 @@
+d2e3f730-df39-42ee-81a839bfb4d024c2 irq_mt818x
+eba8d51f-7827-47b5-82eede6e7743af67 keyword
+d8218443-5ff3-4a4c-b3886cfe07b9562e kpb
+a8a0cb32-4a77-4db1-85c753d7ee07bce6 kpb4
+e50057a5-8b27-4db4-bd799a639cee5f50 kpb_task
+54cf5598-8b29-11ec-a8a30242ac120002 lib_manager
+4f9c3ec7-7b55-400c-86b3502b4420e625 ll_sched
+9f130ed8-2bbf-421c-836ad5269147c9e7 ll_sched_lib
+37f1d41f-252d-448d-b9c41e2bee8e1bf1 main_task
+d23cf8d0-8dfe-497c-82025f909cf72735 math_power
+0cd84e80-ebd3-11ea-adc10242ac120002 maxim_dsm
+425d6e68-145c-4455-b0b2c7260b0600a5 mem
+df5e94d7-fd93-42e9-bb94ab40becc7151 memif
+db10a773-1aa4-4cea-a21f2d57a5c982eb mfcc
+dd400475-35d7-4045-ab030c34957d7a08 micfil
+bc06c037-12aa-417c-9a9789282e321a76 mixer
+39656eb2-3b71-4049-8d3ff92cd5c43c09 mixin
+3c56505a-24d7-418f-bddcc1f5a3ac2ae0 mixout
+ee2585f2-e7d8-43dc-90ab4224e00c3e84 modules
+bb2aa22e-1ab6-4650-85016e67fcc04f4e mtrace_task
+0d9f2256-8e4f-47b3-8448239a334f1191 multiband_drc
+c607ff4d-9cb6-49dc-b6787da3c63ea557 mux
+64ce6e35-857a-4878-ace8e2a2f42e3069 mux4
+1fb15a7a-83cd-4c2e-8b324da1b2adeeaf notifier
+7ae671a7-4617-4a09-bf6d9d29c998dbc1 ns
+376b5e44-9c82-4ec2-bc8310ea101af88f passthrough
+64a794f0-55d3-4bca-9d5b7b588badd037 passthru_smart_amp
+4e934adb-b0ec-4d33-a086c1022f921321 pipe
+f11818eb-e92e-4082-82a3dc54c604ebb3 pipe_task
+d7f6712d-131c-45a7-82ed6aa9dc2291ea pm_runtime
+76cc9773-440c-4df9-95a872defe7796fc power
+9d1fb66e-4ffb-497f-994b17719686596e probe
+7cad0808-ab10-cd23-ef4512ab34cd56ef probe4
+2f0b1901-cac0-4b87-812ff2d5e4f19e4a probe_task
+5c7ca334-e15d-11eb-ba800242ac130004 rtnr
+5276b491-5b64-464e-8984dc228ef9e6a1 sa
+9302adf5-88be-4234-a0a7dca538ef81f4 sai
+3dee06de-f25a-4e10-ae1fabc9573873ea schedule
+70d223ef-2b91-4aac-b444d89a0db2793a sdma
+55a88ed5-3d18-46ca-88f10ee6eae9930f selector
+32fe92c1-1e17-4fc2-9758c7f3542e980a selector4
+cf90d851-68a2-4987-a2de85aed0c8531c sgen_mt8186
+99316bd9-07b9-4665-81796e048d67cb45 sgen_mt8188
+9eb1a55b-fc20-4442-96131ff1023be493 sgen_mt8195
+dabe8814-47e8-11ed-a58bb309974fecce shmread
+e2b6031c-47e8-11ed-07a97f801b6efa6c shmwrite
+167a961e-8ae4-11ea-89f1000c29ce1635 smart_amp_test
+4abd71ba-8619-458a-b33f160fc0cf809b spdai
+a417b6fb-459d-4cf9-be65d38dc9057b80 spi_completion
+9d346d98-203d-4791-baee1770a03d4a71 spinlock
+c1c5326d-8390-46b4-aa4795c3beca6550 src
+e61bb28d-149a-4c1f-b70946823ef5f5ae src4
+33441051-44cd-466a-83a3178478708aea src_lite
+eb0bd14b-7d5e-4dfa-bbe27762adb279f0 swaudiodai
+dd511749-d9fa-455c-b3a713585693f1af tdfb
+04e3f894-2c5c-4f2e-8dc1694eeaab53fa tone
+42f8060c-832f-4dbf-b24751e961997b34 up_down_mixer
+b77e677e-5ff4-4188-af14fba8bdbf8682 volume
+8a171323-94a3-4e1d-afe9fe5dbaa4c393 volume4
+1028070e-04e8-46ab-8d8110a0116ce738 wait
+d944281a-afe9-4695-a043d7f62b89538e waves
+13c8bc59-c4fa-4ad1-b93ace97cd30acc7 wdt
+300aaad4-45d2-8313-25d05e1d6086cdd1 zephyr
+5f1ec3f8-faaf-4099-903ccee98351f169 zephyr_idc
+8fa1d42f-bc6f-464b-867f547af08834da zipc_task
+1547fe68-de0c-11eb-84613158a1294853 zll_sched
```

Comment:
I was thinking about sorting it by UUIDs - then of course any duplication attempts would be immediately obvious, but that is checked by your scripting anyway, so probably sorting by name is better indeed